File: pricing/get-voice-country/get-voice-country.js
// Download the Node helper library from twilio.com/docs/node/install
// These vars are your accountSid and authToken from twilio.com/user/account
var accountSid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX';
var authToken = "{{ auth_token }}";
var PricingClient = require('twilio').PricingClient;
var client = new PricingClient(accountSid, authToken);

client.voice.countries('EE').get(function(err, country) {
    var cpList = country.inboundCallPrices;
    for (var i = 0; i < cpList.length; i++) {
        console.log(cpList[i].numberType + " " + cpList[i].currentPrice + "\n");
    }
});

File: pricing/get-voice-country/get-voice-country.py
# Download the Python helper library from twilio.com/docs/python/install
from twilio.rest import TwilioPricingClient

# Your Account Sid and Auth Token from twilio.com/user/account
account_sid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
auth_token = "your_auth_token"
client = TwilioPricingClient(account_sid, auth_token)

country = client.voice.countries.get("EE")

for p in country.inbound_call_prices:
    print("{} {}".format(p['number_type'], p['current_price']))
